src/components/NewDatePicker/CalendarPicker/ScreenSlideAnimation.js

```diff
@@ -0,0 +1,75 @@
+import {Portal} from '@gorhom/portal';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Dimensions} from 'react-native';
+import Animated, {runOnJS, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+import CONST from '../../../CONST';
+import styles from '../../../styles/styles';
+import variables from '../../../styles/variables';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../withWindowDimensions';
+
+const propTypes = {
+    /** Children to render */
+    children: PropTypes.node.isRequired,
+
+    /** TestID for test */
+    testID: PropTypes.string,
+
+    ...windowDimensionsPropTypes,
+};
+
+const defaultProps = {
+    testID: undefined,
+};
+
+const ScreenSlideAnimation = React.forwardRef(({children, testID, isSmallScreenWidth}, ref) => {
+    const [visible, setVisible] = React.useState(false);
+    const width = isSmallScreenWidth ? Dimensions.get('window').width : variables.sideBarWidth;
```

Comment:
Why not use `windowWidth` from `withWindowDimensions` hook?

Reply:
Good catch!